test/cne/cne.go

```diff
@@ -0,0 +1,33 @@
+package cne
```

Comment:
The package name does not match what's in the Makefile:
`SUITE=./test/cloud-event-proxy `

Reply:
Thanks, Fixed